Module-10/CatFacts.playground/Contents.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import PlaygroundSupport
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+// 📌 Requirement:
+// ✅ A struct for coding the JSON response from https://catfact.ninja/fact
+
+struct CatFact: Codable {
+    let fact: String
+    let length: Int
+}
+
+extension CatFact {
+    /// The URL of the catfact.ninja API used to retrieve a random cat fact.
+    private static let endpointURL = URL(string: "https://catfact.ninja/fact")!
+
+    /// Retrieves a random CatFact from catfact.ninja using URLSession.shared.
+    static func retrieveFact() async throws -> CatFact {
+        let (data, _) = try await URLSession.shared.data(from: endpointURL)
+        return try JSONDecoder().decode(CatFact.self, from: data)
```

Comment:
Because I was lazy! 😄 
I have refactored networking into a CatFactService with the ability to change endpoints.  This should also help with any unit testing.

See commit 4d45c87 .